### PR TITLE
Set FORK_BLOCK in EIP-2929 and EIP-2930

### PR DESCRIPTION
**Pull request by paulrberg**
Both EIP-2929 and EIP-2930 have been added in the Berlin hard fork.

### CODE REVIEW

**Review by Pandapip1 — APPROVED** [review]
LGTM

**Review by MicahZoltu — APPROVED** [review]

**Comment by MicahZoltu** [comment]
I'm going to take a pretty big risk here and merge this without waiting for author approval, since I'm quite confident that @holiman/@vbuterin would approve.  If they come back after merging saying they don't want this, I'll revert the change.

**Pull request merged**
